modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -85,6 +85,37 @@ Note: If the payload is used as part of the message log, do not configure the Li
 |After Calling API |Set to cause your logging policy to log messages after the API endpoint is called, taking into account the order in which this policy is applied |Optional
 |===
 
+== Example of Logging Policy from the UI
+
+To better understand how to use the Message Logging policy, consider an example "Les Vetments" company that is in the business of retail clothing and more. To run their e-commerce portal successfully, Les Vetments needs to audit every event and transaction made on their website.
+
+For example, if a customers buys or returns a product, or if the company adds or removes a product from the catalogue, the transactions must be logged. The IT manager at Les Vetments applies the Message Logging policy to their APIs to enable logging and viewing any data or transaction changes without making any modifications in the code.
+
+Les Vetments uses the resource-level policy's granularity to apply the policy to only `POST`, `PUT`, `PATCH` and `DELETE` methods. To log all events, Les Vetments applies the Message Logging policy to the API with the following UI configurations:
+
+*Message*: #[payload]
++
+Everything in the payload must be logged.
+*Conditional*: Blank
++
+Because Les Vetments requires all types of messages, no Mule expression ito filter messages is specified.
+*Category*: Blank
++
+No prefix is required in the log sentence.
+*Level*: Info
++
+All types of messages are to be logged: Info, Warn, Error, or Debug.
+*Before Calling API*: checked
++
+All transactions occurring before the API is called is to be logged.
+*After calling API*: Checked
++
+All transactions occurring after the API is called is to be logged.
+
+Now, whenever a customer accesses the Les Vetments catalogue and adds or removes items from their cart, or when an item is added or removed from the catalogue, logs are generated:
+//add the output example here
```

Comment:
Can you try again creating an API from scratch and applying the policy with this configuration to see if you can see the output now?

Reply:
Done! Thanks!